### PR TITLE
create mat dialog in design system

### PR DESCRIPTION
**Pull request by bhushankhope**

### CODE REVIEW

**Review by LibbyUX — DISMISSED** [review]
Thank you @bhushankhope !

File: libs/design-system/dialog/src/lib/notice/notice.component.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  .mat-mdc-dialog-title::before {
+    display: none;
+  }
+
+  .mat-mdc-dialog-content {
+    --mdc-dialog-supporting-text-tracking: 0.5px;
+    padding: 0 2rem;
+    font: var(--sys-body-large);
```
Comment (axdanbol) [inline libs/design-system/dialog/src/lib/notice/notice.component.scss:35]: Missing letter-spacing

File: libs/design-system/dialog/src/lib/notice/notice.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    const dismiss = await screen.findByRole('button', { name: 'Dismiss' });
+    expect(dismiss).toBeInTheDocument();
+  });
+
+  it('Action button should be visible', async () => {
+    service.openNotice(TITLE, MESSAGE, {
+      label: 'Action Button',
+      callback: () => {
+        ('');
```
Comment (axdanbol) [inline libs/design-system/dialog/src/lib/notice/notice.component.spec.ts:40]: Woot! Better practice it to use jest.fn() when creating empty functions. By doing so you can also test that the passed function is called when the action button is clicked.
```ts
const callback = jest.fn();
service.openNotice(..., {
   ...,
   callback
});

...

await user.click(dismiss)

expect(callback).toHaveBeenCalledTimes(1);
```

File: libs/design-system/dialog/src/lib/notice/notice.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      },
+    });
+
+    const dismiss = await screen.findByRole('button', { name: 'Action Button' });
```
Comment (axdanbol) [inline libs/design-system/dialog/src/lib/notice/notice.component.spec.ts:44]: Rename variable to `actionButton` or something similar.

**Review by axdanbol — CHANGES_REQUESTED** [review]
A few minor changes

File: libs/design-system/dialog/src/lib/notice/notice.component.scss (hunk 1 above)
Comment (bhushankhope) [reply libs/design-system/dialog/src/lib/notice/notice.component.scss:35]: --mdc-dialog-supporting-text-tracking: 0.5px;
is letter spacing

**Review by axdanbol — APPROVED** [review]

**Pull request merged**